src/MiNET/MiNET/Player.cs

```diff
@@ -1189,7 +1317,7 @@ public virtual void Teleport(PlayerLocation newPosition)
 				Monitor.Exit(_teleportSync);
 			}
 
-			MiNetServer.FastThreadPool.QueueUserWorkItem(SendChunksForKnownPosition);
+			//MiNetServer.FastThreadPool.QueueUserWorkItem(SendChunksForKnownPosition);
```

Comment:
Is there a reason for not doing this anymore?

Reply:
Chunks are often sent before the client can process them, because of this they are lost.
The most correct solution is to send them only after teleportation. After that, they will not go, since MINET thinks that the player has already received them.